### ISSUE TITLE
Have you used this code to verify the experiment in the paper? How is the result compared to the paper?

### ISSUE THREAD

**Issue by xuequeen** [apple2373/metapath2vec#3]
Have you used this code to verify the experiment in the paper? How is the result compared to the paper?

**Comment by apple2373** [comment]
no. i didn't do the reproducing experiment. I recommend author's own
implementation if you are looking for verified implementation.

**Comment by 960924** [comment]
> Have you used this code to verify the experiment in the paper? How is the result compared to the paper?

Have you done the relevant experiment? I used the node representation generated by the author's code, but the classification effect is only about 0.6. I don't know what is wrong.
